core/runtime/jsruntime.rs

```diff
@@ -648,6 +652,7 @@ impl JsRuntime {
       // SAFETY: we just asserted that layout has non-0 size.
       unsafe { std::alloc::alloc(layout) as *mut _ };
 
+    let custom_module_evaluation_cb = options.custom_module_evaluation_cb;
     let validate_import_attributes_cb = options
```

Comment:
Discussed with @lucacasonato and we decided we should remove it. It should be up to embedder's `ModuleLoader` to deny loading a module on unknown attributes. Upstream in a separate PR.

Reply:
https://github.com/denoland/deno_core/pull/435